tests/rt_coastal.conf

```diff
@@ -21,26 +21,57 @@
 ### Intel Tests ###
 ### COASTAL ###
 
-# ADCIRC tests
+##########
+### ADCIRC tests: 1-5
 COMPILE | 1 | intel | -DAPP=CSTLA -DADCIRC_CONFIG=PADCIRC -DCOUPLED=ON | | fv3 |
 RUN | coastal_florence_hsofs_atm2adc | | baseline |
 RUN | coastal_ike_shinnecock_atm2adc | | baseline |
 
-# FVCOM tests
-COMPILE | 2 | intel | -DAPP=CSTLF -DCOORDINATE_TYPE=SPHERICAL -DWET_DRY=ON | | fv3 |
+#P.V COMPILE | 2 | intel | -DAPP=CSTLAW -DADCIRC_CONFIG=PADCIRC -DCOUPLED=ON | | fv3 |
+#P.V RUN | coastal_florence_hsofs.atm2adc2ww3 | | baseline |
+#P.V RUN | coastal_ike_shinnecock.atm2adc2ww3 | | baseline |
```

Comment:
Why we have bunch of test that are commented out. I know we are playing to bring all the cases from CoastalApp test suite to here but it would be add them one by one when they are available.

Reply:
I have the tests commented out because most of them are not implemented/tested yet. Why is this a problem at this stage?